xarray/core/accessors.py

```diff
@@ -147,3 +184,21 @@ def f(self, dtype=dtype):
     time = _tslib_field_accessor(
         "time", "Timestamps corresponding to datetimes", object
     )
+
+    def _tslib_round_accessor(self, name, freq):
+        obj_type = type(self._obj)
+        result = _round_field(self._obj.data, name, freq)
+        return obj_type(result, name=name,
+                        coords=self._obj.coords, dims=self._obj.dims)
+
+    def floor(self, freq):
+        ''' Round timestamp downward. '''
```

Comment:
It would be nice to have a docstring that explains what the freq argument should be.